spec.html

```diff
@@ -33975,7 +33975,7 @@ <h1>
             <dd>For the purposes of this abstract operation, a <em>decimal digit</em> is a code unit in the range 0x0030 (DIGIT ZERO) to 0x0039 (DIGIT NINE) inclusive.</dd>
           </dl>
           <emu-alg>
-            1. Let _stringLength_ be the number of code units in _str_.
+            1. Let _stringLength_ be the length of _str_.
```

Comment:
There's a handful of other uses of "the number of code units in". If we're decided that we prefer "length" (which I personally do), we should fix those as well, in a separate PR.

Reply:
#2746